indico/modules/events/surveys/models/surveys.py

```diff
@@ -196,3 +229,21 @@ def close(self):
         if self.state not in (SurveyState.active_and_clean, SurveyState.active_and_answered):
             raise IndicoError("Survey can't be closed")
         self.end_dt = now_utc()
+
+    def send_start_notification(self):
+        if not self.notifications_enabled or self.start_notification_sent:
+            return
+        addresses = self.start_notification_emails
```

Comment:
Sure, however in that case I need to convert every list to a set. Isn't it simpler just to convert the result list to a set?